api/namex/resources/mras.py

```diff
@@ -109,6 +109,7 @@ def get(self, province, corp_num):
                 headers=headers
             )
 
+
```

Comment:
'Accept': 'application/json' header does not work with MRAS API, as it is ignored (unimplemented by the provider of service), so probably needs to switch to xml, just in case provider does not decide to implement json